fish_right_prompt.fish

```diff
@@ -24,6 +23,15 @@ function fish_right_prompt
       end
     end
 
-    echo -n -s $ruby_info $normal
+    if [ "$theme_display_rbenv_with_gemfile_only" = 'yes' ]
+      if test -f Gemfile
+        echo -n -s $ruby_info $normal
+      end
+      return
```

Comment:
Hey thanks for that. It was not doing anything wrong, but with the `return` call and the lack of `else`, any future modification would be impacted by that. The person doing the change would lose sometime to understand why his code was not being called until he noticed the `return` call was preventing any future call from being done.

:+1: